blog/2021-12-20-cwa-2-15/index.md

```diff
@@ -0,0 +1,92 @@
+</center>
+<br></br>
+
+The Corona-Warn-App checks the provider's requirements and shows the user only those certificates for selection that meet the requirements. 
+
+For example, if a provider requests either a vaccination certificate, a recovery certificate, a rapid test certificate, or a PCR test certificate, the user will be shown all of these certificates to choose from, provided they have stored them in the app. If a provider requires either a vaccination certificate or a recovery certificate, the app will only provide these certificates to choose from. If the user does not have a certificate that meets the requirements, the CWA will notify them accordingly.  
+
+
+<br></br>
+<center> 
+<img src="./ticket-validation(1).png" title="Selection of certificate in the app" style="align: center" width=250> 
+</center>
+<br></br>
+
+After the user has selected a certificate and once again given their consent, the certificate is checked. For this purpose, it is temporarily transmitted in encrypted form to the validation service. The validation service then compares the organizer's rules with the certificate and then informs the booking system whether the certificate complies with the rules or not. 
+
+<br></br>
+<center> 
+<img src="./ticket-validation(7).png" title="Consent for temporary transmission of certificate" style="align: center" width=250> 
+</center>
+<br></br>
+
+ 
+Please notice: the certificates remain on the user's own smartphone and are not stored in the validation system at any time. 
+
+The user can then see in the app whether the certificate was successfully verified or whether it was not recognized.
+
+
+<br></br>
+<center> 
+<img src="./ticket-validation(6).png" title="verification successful" style="align: center" width=250> <img src="./ticket-validation(5).png" title="certificate not recognized" style="align: center" width=250>
+</center>
+<br></br>
+
+A certificate can be "not recognized" if it does not meet the provider's requirements. This may be the case, for example,  ifthe user has selected an antigen test as proof that is older than 48 hours. 
+
+### What should users be aware of?
+
+Users should make sure that the name in the certificate they want to use for verification is EXACTLY the same as the name they entered in the booking system. If this is not the case, they will not be able to select the corresponding certificate in the Corona-Warn-App. 
+
+For example, if users have entered their first and middle names in their certificate, they should also enter both names when booking tickets. If they only enter one name, the CWA will not be able to recognize that it is the same person. 
+
+In addition, with version 2.15, the project team has adapted the guidelines under a "red warning" (warning about an encounter with increased risk). Detailed information on how users should act in case of a red tile can be found in this blog: [link]

```

Comment:
@dsarkar the link to the blog is missing here and in the German blog.

Reply:
@Ein-Tim Thanks. Was fixed by HannaHei in the meantime.